axelrod/plot.py

```diff
@@ -59,3 +61,33 @@ def payoff(self):
             return figure
         else:
             return None
+
+    def stackplot(self, populations):
+        if self.matplotlib_installed:
+            figure, ax = plt.subplots()
+            turns = range(len(populations))
+            pops = [[populations[iturn][ir] for iturn in turns] for ir in self.result_set.ranking]
+            ax.stackplot(turns, *pops)
```

Comment:
I think it could be nicer to sort over the final populations. Could be (especially when we include possibility of mutation) that this does not coincide with the same order.